Fhi.HelseId.Common/Identity/Jwt.cs

```diff
@@ -14,33 +14,31 @@ public static class ClientAssertion
     {
         public static string Generate(string clientId, string authority, SecurityKey securityKey)
         {
-            var signingCredentials = new SigningCredentials(securityKey, SecurityAlgorithms.RsaSha512);
+            var signingCredentials = new SigningCredentials(securityKey, SecurityAlgorithms.RsaSha256);
```

Comment:
Hva er bakgrunnen for denne endringen?

Reply:
bra funnet. Jeg har nå lagt inn if UseIdporten